CMakeLists.txt

```diff
@@ -559,6 +562,10 @@ if(MSVC)
   set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -Xcompiler  /Zc:__cplusplus")
 
   set(CMAKE_NINJA_CMCLDEPS_RC OFF)
+  if(MSVC_Z7_OVERRIDE)
+    # CMake set debug flags to use /Z7
+    set(CMAKE_MSVC_DEBUG_INFORMATION_FORMAT Embedded)
```

Comment:
Could this cause lib size increase between 06.14 and 06.15 ?
<img width="1430" height="584" alt="Screenshot 2025-07-30 at 4 17 05 PM" src="https://github.com/user-attachments/assets/f940ce18-a95a-4909-a2f1-ffdb6d4ad746" />

Reply:
Very likely, let's propose a change to fix it.